frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/PipelineRunDrawerRightTabs.tsx

```diff
@@ -29,7 +29,7 @@ const PipelineRunDrawerRightTabs: React.FC<PipelineRunDrawerRightTabsProps> = ({
   taskReferences,
   parameters,
 }) => {
-  const [selection, setSelection] = React.useState(PipelineRunNodeTabs.INPUT_OUTPUT);
+  const [selection, setSelection] = React.useState(PipelineRunNodeTabs.LOGS);
```

Comment:
@yannnz I have made Logs tab to the default one. Logs will only load, when it is a default. Let me know whether this change is fine.

Reply:
> I have made Logs tab to the default one. Logs will only load, when it is set as default. Let me know whether this change is fine.

cc @yih-wang

Reply:
I didn't quite follow that. Do you mean the log will only load when the Logs tab is selected?

Reply:
we were showing the INPUT_OUTPUT tab as the default one when we opened the Drawer. But I changed that to the LOGS tab to load the logs.

Reply:
Not selecting the first tab by default doesn't seem intuitive to me. I'll let @yannnz chime in here. Does the Input/Output tab have content before the run is completed?

Reply:
It will show the "input parameters or No content" when it is running. 

[Screencast from 2023-11-22 13-30-30.webm](https://github.com/opendatahub-io/odh-dashboard/assets/99261071/f0ce8215-4728-4537-8e7c-287496e7a353)

Reply:
> Not selecting the first tab by default doesn't seem intuitive to me. I'll let @yannnz chime in here. Does the Input/Output tab have content before the run is completed?

I don't get the reason for not defaulting the first tab. This doesn't seem a common practice unless we provide a function to remember the tab user selected last time when opening the drawer with a solid reason.

Reply:
I agree with @yannnz this shouldn't be done. Especially not because of a technical reason. I'll have to look into this when I get a moment. You likely need to control the mount of the tab resource. There is definitely failure points here we need to look at with the PF Log Viewer. But that shouldn't impact the UX

Reply:
Is it okay to use the conditional rendering for tab like this?
```
<TabContent {...tabContentProps(PipelineRunNodeTabs.LOGS)}>
          {selection == PipelineRunNodeTabs.LOGS && 
            <LogsTab task={task} />
          }
 </TabContent>
```

Reply:
We had rendering issues with PF Logviewer(which is why we are using the Monaco editor), to fix the rendering issues, this PR has set the `Logs` tab as the default tab and it seems to be working fine(we don't get rendering issues-rendering issues=the logs don't load or keeps re-rendering unless we reach the end of logs). However, the `Logs` tab cannot be the default tab. So ideally this is not the solution for the rendering issues we are facing with PF Logviewer.

Reply:
I'm confused... I saw the logs render when I reset the value 🤔 What Pipeline, what are you seeing? I'm aware you guys are noting an issue with the logs, but I'm not sure I have seen it anywhere... what exactly is happening? Can I get a gif of the scenario? Is there one here and I'm missing it?

Reply:
For posterity, I investigated and found the root cause of this rendering issue in LogViewer. I opened an issue in PF here: https://github.com/patternfly/react-log-viewer/issues/55.

I'll open a PR against this branch with the workaround we discussed [on Slack here](https://redhat-internal.slack.com/archives/C05SMJ09DD2/p1701702085040869).